namer/namer.cc

```diff
@@ -359,16 +351,6 @@ class NameInserter {
                         e.setHeader("Superclasses must only contain constant literals");
                     }
                     anc = ast::MK::EmptyTree();
-                } else if (shouldLeaveAncestorForIDE(anc) &&
-                           (klass->kind == ast::Module || anc != klass->ancestors.front())) {
-                    klass->rhs.emplace_back(ast::MK::KeepForIDE(anc->deepCopy()));
-                }
-            }
-        }
-        if (!klass->singletonAncestors.empty()) {
-            for (auto &sanc : klass->singletonAncestors) {
-                if (shouldLeaveAncestorForIDE(sanc)) {
-                    klass->rhs.emplace_back(ast::MK::KeepForIDE(sanc->deepCopy()));
```

Comment:
I like that we don't have to have these shouldLeaveAncestorForIDE calls in the tree anymore.